src/utils.cpp

```diff
@@ -5819,6 +5836,74 @@ struct obj_data *make_new_finished_program(int part_type, int mpcp, int rating=0
   return prog;
 }
 
+struct obj_data *make_otaku_deck(struct char_data *ch) {
+  if (!ch) {
+    mudlog_vfprintf(ch, LOG_FUCKUPLOG, "Tried to create an otaku deck, but provided invalid character.");
```

Comment:
`LOG_SYSLOG` please, the fuckuplog is exclusively for players writing invalid commands that I need to add to the `fix_common_fuckups()` function later.